### PR TITLE
[ja] Translate Gateway API into Japanese

### PR DESCRIPTION
**Pull request by tamilselvan1102**
fixes : #47680

### CODE REVIEW

**Comment by tamilselvan1102** [comment]
page preview : https://deploy-preview-47728--kubernetes-io-main-staging.netlify.app/ja/docs/reference/glossary/?all=true#term-gateway-api

**Comment by Okabe-Junya** [comment]
/retitle [ja] Translate Gateway API into Japanese

File: content/ja/docs/reference/glossary/gateway.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: ゲートウェイ API（Gateway API）
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/gateway.md:2]:
```suggestion
title: Gateway API
```

File: content/ja/docs/reference/glossary/gateway.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+short_description: >
+  Kubernetesでサービスネットワークをモデル化するために使用されるAPI。
+
+aka:
+tags:
+- networking
+- architecture
+- extension
+---
+Kubernetesでサービスネットワークをモデル化するために使用されるAPIクラスのセット。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/gateway.md:15]:
```suggestion
Kubernetesでサービスネットワーキングをモデル化するために使用されるAPIクラス群。
```

File: content/ja/docs/reference/glossary/gateway.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+<!--more--> 
+Gateway APIはKubernetesでサービスネットワークをモデル化するための拡張可能、ロール指向、プロトコル認識APIクラスのセットを提供します。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/gateway.md:18]:
```suggestion
Gateway APIは、Kubernetesにおけるサービスネットワーキングをモデル化するための、拡張性が高く、ロール指向で、プロトコル対応のAPIクラス群を提供します。
```

**Review by Okabe-Junya — CHANGES_REQUESTED** [review]
Thanks, I have some comments

**Comment by Okabe-Junya** [comment]
/assign nasa9084 
for approval
/label tide/merge-method-squash

**Review by Okabe-Junya — COMMENTED** [review]
/lgtm

**Pull request merged**